.travis.yml

```diff
@@ -4,19 +4,27 @@ python:
   - 3.3
   - 3.4
   - 3.5
-#  - 3.6
+  - 3.6
 env:
   - DJANGO=1.7
   - DJANGO=1.8
   - DJANGO=1.9
-#  - DJANGO=1.10
-#  - DJANGO=1.11
+  - DJANGO=1.10
+  - DJANGO=1.11
 matrix:
   exclude:
+    # Django 1.7 does not support Python 3.5+
     - python: 3.5
       env: DJANGO=1.7
+    - python: 3.6
+      env: DJANGO=1.7
+    # Django 1.9+ does not support Python 3.3
```

Comment:
You don't need to add those comments. The data should be self-explanatory (`exclude` section. Python + Django versions specified). -- Comments are usually an indicator that the code or configuration is not self-explantory.

Reply:
Hmm, it's not self-explanatory to me: I had to go look up on the Django docs which combinations are actually supported (which is why I added the comments).

Reply:
Well, that's normal. Actually, the docs could err. I prefer to look at the PyPI package meta data.

I would prefer to keep the Travis configuration clean. As I said, you would otherwise have to explain why Python 3.2 is missing.

Reply:
I don't want to be the dog in the manger:

If you really care that much you may add a general comment above the `exclude:` section, kind of "Exclude unsupported Python/Django combinations", or so. If it's really necessary, in the end.

Reply:
The sections specify the versions, so I think one comment is exactly the right number; no comment and I begin to wonder why we exclude these. This does not block the build for me, though.

Reply:
@jcassee: The reason I made both comments is to explicitly document the different version ranges that are known to be broken (only Django 1.7 with any Python 3.5+, versus only Python 3.3 with any Django 1.9+).

Having only a list of existing version numbers doesn't make it obvious what ranges they are intended to represent, and doesn't give any guidance on how exactly how new Django and Python releases should be added to the list, or not. The comments hopefully make this more obvious. (They're certainly useful to me, for future maintenance.)